src/backend/controllers/dashboard-controller.js

```diff
@@ -1,7 +1,11 @@
+/* eslint-disable no-unused-vars */
 const BaseController = require('./base-controller.js')
 
 class DashboardController extends BaseController {
-  index({params, query, payload}, response) {
+  async index({ params, query, payload }, response) {
+    // eslint-disable-next-line new-cap
+    const dashboard = this._admin.dashboardPage && new this._admin.dashboardPage(this._admin)
```

Comment:
dashboardPage should be capitalized: this._admin.DashboardPage - its a class not an instance